.circleci/config.yml

```diff
@@ -0,0 +1,86 @@
+version: 2
+
+main: &main
+    machine: true
+    environment:
+      # Used by qthelpers to close widgets after a defined time
+      - TEST_CI: "True"
+      - TEST_CI_APP: "True"
+    steps:
+      - checkout
+      - run:
+          command: docker pull dorowu/ubuntu-desktop-lxde-vnc:trusty
+      - run:
+          name: Install system packages
+          command: |
+            sudo apt-get update
+            sudo apt-get install libegl1-mesa
+      - run:
+          name: Install Miniconda
+          command: |
+            wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+            bash miniconda.sh -b -p $HOME/miniconda
+            export PATH="$HOME/miniconda/bin:$PATH"
+            conda install python=$PYTHON_VERSION
```

Comment:
In this step:

1. Please move these commands to an `install.sh` script in `.circleci`
2. Please use Astropy's `ci-helpers` instead of installing Miniconda by hand. That's what we have adopted is most other repos in our org. See for example qtpy's [install.sh](https://github.com/spyder-ide/qtpy/blob/master/.circleci/install.sh).